homeassistant/components/image_processing/opencv.py

```diff
@@ -7,22 +7,56 @@
 from datetime import timedelta
 import logging
 
+import requests
+
+import voluptuous as vol
+
 from homeassistant.core import split_entity_id
 from homeassistant.components.image_processing import (
-    ImageProcessingEntity, PLATFORM_SCHEMA)
-from homeassistant.components.opencv import (
-    ATTR_MATCHES, CLASSIFIER_GROUP_CONFIG, CONF_CLASSIFIER, CONF_ENTITY_ID,
-    CONF_NAME, process_image)
+    CONF_SOURCE, CONF_ENTITY_ID, CONF_NAME, PLATFORM_SCHEMA,
+    ImageProcessingEntity)
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['opencv-python==3.2.0.6', 'numpy==1.12.0']
```

Comment:
Is it possible to remove the opencv-python here and check the import error on platform load? On ARM/AARCH64 I compile opencv into image but it will not work while this pip packages not install able on thi arch. Or had that change?